### ISSUE TITLE
Prettier Button/Shortcut does not work

### ISSUE THREAD

**Issue by phillihoch** [expo/snack#291]
### Summary

When I open a fresh snack and I make some tabs in the code, so the Prettier-Function should format it, the Prettier-Function does not work.
Browser console shows that error
![image](https://user-images.githubusercontent.com/64873861/168115399-91e2e533-4dc7-427f-a360-613d9768521a.png)

### What platform(s) does this occur on?

Android, iOS, Web

### SDK Version

44

### Reproducible demo or steps to reproduce from a blank project

- Open new snack
- Format code that it does not look formatted (make some tabs in some lines)
- Press "Prettier"

**Comment by MehmetKaplan** [comment]
An atomic example to reproduce is as follows.

Problematic version:
```javascript
import * as React from 'react';
import { StyleSheet } from 'react-native';

function App() {}

const styles = StyleSheet.create({}); // removing this line, Prettier becomes prettier.

export default App;
```

non-Problematic version:
```javascript
import * as React from 'react';
import { StyleSheet } from 'react-native';

function App() {}

export default App;
```

What I see in the browser's console:

<img width="803" alt="image" src="https://user-images.githubusercontent.com/2387157/168225765-294709b8-7c34-4227-9e53-d5e238d8bd28.png">

Pasting the text version also:
```javascript
24.fa51bb4c2ad05086ef69.chunk.js:1 Uncaught (in promise) ReferenceError: G is not defined
    at 24.fa51bb4c2ad05086ef69.chunk.js:1:226036
    at t.exports.call (24.fa51bb4c2ad05086ef69.chunk.js:1:111449)
    at 24.fa51bb4c2ad05086ef69.chunk.js:1:226028
    at 24.fa51bb4c2ad05086ef69.chunk.js:1:226185
    at t.exports.call (24.fa51bb4c2ad05086ef69.chunk.js:1:111449)
    at t.exports (24.fa51bb4c2ad05086ef69.chunk.js:1:225772)
    at printCallExpression (24.fa51bb4c2ad05086ef69.chunk.js:1:229069)
    at 24.fa51bb4c2ad05086ef69.chunk.js:1:284148
    at Object.print (24.fa51bb4c2ad05086ef69.chunk.js:1:288758)
    at l (24.fa51bb4c2ad05086ef69.chunk.js:1:114441)
```

**Comment by phillihoch** [comment]
Okay, thank you.
That works just fine :)

But I think it would be good to fix the problem because StyleSheet.create() has some advantages in terms of performance. For developing an app its okay, but for production that should be fixed.

**Comment by MehmetKaplan** [comment]
Sorry I guess there may be a misunderstanding. The 2 atomic examples are just snack codes where prettier fails and not fails respectively. (For expo snack editor’s developers to evaluate.)

StyleSheet, what happens in production, etc are irrelevant with prettier’s fail in the expo snacks editor. 

PS: If you mean I was suggesting you to remove the StyleSheet, I was not. :-)

**Comment by phillihoch** [comment]
No all good :)

I removed it to pretty print. Interesting to mention: in some cases the code formats now without the StyleSheet. In other cases it still behaves the same. So not only the StyleSheet creates that error.

For now I am removing the stylesheet to have the pretty print. I'm working on a pretty small snack right now, so it's more effort to format the code by hand than removing the Stylesheet and adding it again ;)

**Comment by byCedric** [comment]
Hi! Sorry for the silence from our end. I think I found out why this is happening, but I have no fix yet. 
- When running the Snack website in development mode, it seems to work fine. Comments, code, and all languages are prettier-able
- When running the Snack website in production mode, it seems that some parts within prettier are undefined.

I fiddled around in our Webpack config and it seems that the `terser-webpack-plugin` is minifying already minified code, specifically all prettier modules used in **[utils/prettierCode.tsx](https://github.com/expo/snack/blob/main/website/src/client/utils/prettierCode.tsx)**. Unfortunately, [Terser doesn't really support excluding node modules to minify](https://github.com/webpack-contrib/terser-webpack-plugin/issues/31#issuecomment-442202331). 

So we need to [swap this part out](https://github.com/expo/snack/blob/main/website/webpack.config.js#L40-L45) for something that either does not break prettier, use a non-minified version of prettier, or a terser alternative that allows prettier to not be double-minified.

PRs welcome :)
